src/components/common/YourCompany/BusinessContactInfo.vue

```diff
@@ -92,10 +92,11 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
    * Use "immediate" to pick up all validity conditions
    */
   @Watch('isEditingContact', { immediate: true })
-  private onIsEditingContact (): void {
+  @Watch('getBusinessContact.email', { immediate: true })
```

Comment:
I forgot to watch this, which is also needed to check validity.

It doesn't need to be "immediate" since this method will fire "immediately" (when this component is created) due to the line above, but this looks consistent.

Note that another way to watch an object property is to watch the object and add `deep: true` to the options here. In this case we only care about email (not all properties) so this is appropriate.